app/views/shared/_header.html.erb

```diff
@@ -0,0 +1,12 @@
+<section class="header">
+  <a href="#" class="text-white" onclick="window.history.back(); return false;">
+    <i class="bi bi-arrow-left-circle" style="font-size: 1.5rem;"></i>
+  </a>
```

Comment:
The back arrow should not behave as the back arrow from our browser, and should not redirect to the previous page.
For example, 
- the home page should not have a back arrow, because we don't want the user to be redirected to the Sign In page again.
- when we are on the Transaction page and after adding a new expense, we don't want the back arrow to take us to the expense form again, but we want to direct us to the Categories index page.

- [ ] Kindly address both these wrong behaviour of the back arrow button.